roles/v2/pool/src/lib/mining_pool/message_handler.rs

```diff
@@ -203,157 +210,4 @@ impl ParseDownstreamMiningMessages<(), NullDownstreamMiningSelector, NoRouting>
         };
         Ok(SendTo::Respond(Mining::SetCustomMiningJobSuccess(m)))
     }
-
```

Comment:
Nice! Don't know why I left it here but it redundant. Only thing the we should add to the function defined in the trait is the below:
```rust
                    (SupportedChannelTypes::Group, true) => self_mutex
                        .safe_lock(|self_| self_.handle_set_custom_mining_job(m))
                        .map_err(|e| roles_logic_sv2::Error::PoisonLock(e.to_string()))?,
``` 
This should be added in the trait definition in the default `handle_message_mining` where we match `Ok(Mining::SetCustomMiningJob(m))`

Reply:
No looked at it better and version of this function that we have in the trait is the right one so no need to add the above. That's cause we can not have a supported channel type that is Group only with work selection enabled is just not possible under Sv2